docs/pages/api-docs/button-base.json

```diff
@@ -0,0 +1,65 @@
+{
+  "description": "`ButtonBase` contains as few styles as possible.\nIt aims to be a simple building block for creating a button.\nIt contains a load of style reset and some focus/ripple logic.",
```

Comment:
Oops, the component descriptions should be excluded. It might be a good time to apply https://github.com/mui-org/material-ui/pull/23580/commits/94af805e6b9879ffe257dc720d25f4f285bac896

Reply:
I don't get it, why not? It's present in https://next.material-ui.com/api/button-base/.

Reply:
They're here: https://github.com/mui-org/material-ui/blob/41543c16a5f221e12fd809dc4069f323f5a54d0c/docs/translations/component-descriptions.json

Reply:
Why do we need `/component-descriptions.json`?

Reply:
Oh, I get it.